src/betterproto/__init__.py

```diff
@@ -1063,11 +1061,13 @@ def from_dict(self: T, value: Dict[str, Any]) -> T:
                         else:
                             v = [cls().from_dict(item) for item in value[key]]
                     elif isinstance(v, datetime):
-                        v = datetime.fromisoformat(value[key].replace("Z", "+00:00"))
+                        v = isoparse(value[key])
                         setattr(self, field_name, v)
                     elif isinstance(v, timedelta):
                         v = timedelta(seconds=float(value[key][:-1]))
                         setattr(self, field_name, v)
+                    elif isinstance(v, bool):
```

Comment:
Is this another sneaky bugfix? Could you add a test case to clarify?

Reply:
Removed this off topic addition